.github/workflows/pythonpackage.yml

```diff
@@ -20,21 +20,25 @@ jobs:
           python-version: ${{ matrix.python-version }}
           architecture: x64
       - name: Install and configure Poetry
-        uses: snok/install-poetry@v1.1.1
+        uses: snok/install-poetry@v1.1.6
         with:
-          version: 1.1.4
-          virtualenvs-create: false
+          version: 1.1.6
```

Comment:
This is unintentionally possibly a little confusing. Just to clarify - the `version` parameter here specifies the Poetry version, while the `@1.1.6` specifies the action version. They just happen to be the same.